openmetadata-ui/src/main/resources/ui/src/pages/AlertDetailsPage/AlertDetailsPage.test.tsx

```diff
@@ -84,6 +84,15 @@ jest.mock('../../utils/ToastUtils', () => ({
   showErrorToast: jest.fn(),
 }));
 
+jest.mock('../../components/PageLayoutV1/PageLayoutV1', () => {
+  return jest.fn(({ children, pageTitle }) => (
+    <div data-testid="page-layout-v1">
+      <h1>{pageTitle}</h1>
+      <div>{children}</div>
+    </div>
+  ));
```

Comment:
Always try to mock functions with `.mockImplementation` for clarity and consistency